MANIFEST.in

```diff
@@ -1,7 +1,20 @@
-include *.py
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
+include *.rst
+include *.txt
 include buildout.cfg
+include tox.ini
 
+exclude MANIFEST.in
+
+recursive-include docs *.bat
+recursive-include docs *.py
+recursive-include docs *.rst
 recursive-include zc *.txt
+recursive-include docs Makefile
 
-global-exclude *.pyc
-global-exclude *.pyo
+recursive-include src *.pt
+recursive-include src *.rst
+recursive-include src *.txt
+recursive-include src *.xml
+recursive-include src *.zcml
```

Comment:
This should probably be changed to `zc` instead of `src`

Reply:
And then remove the duplicate `recursive-include zc *.txt`

Reply:
Thank you!